src/read_data/query_data.R

```diff
@@ -0,0 +1,107 @@
+#####################################
+##### Prerequisites
+# None
+
+##### Data 
+# Argus API
+
+##### Steps
+# Authentification to Argus server
+# Collection of the data of interest
+#
+#
+#
+#
+
+#### Results
+# Dataframes with the needed data for Argus dashboards
+#######################################
+
+library(httr)
+library(jsonlite)
+library(dplyr)
+library(purrr)
+
+############ Needed variables
+# access to the server
+
+main_api_url <- "http://avadar.novel-t.ch/argus-r/sesdashboard/web/"
+user_name <- ""
```

Comment:
it's good to read such variables from environment, so that you don't have to modify source code to run it